index.html

```diff
@@ -0,0 +1,193 @@
+        <h1>Kidde George</h1>
+        <h2>Software Developer</h2>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+</section>
+<div class="img">
+    <img src="images/Vector4.png" alt="line">
+</div>
+<section class="able">
+    <div class="ability">
+        <ul>
+         <li><h4>Languages</h4></li>
+         <li>JavaScript</li>
+         <li>Ruby</li>
+         <li>Html</li>
+         <li>Css</li>
+        </ul>
+        <ul>
+         <li><h4>Frameworks</h4></li>
+         <li>Bootstrap</li>
+         <li>Ruby on Rails</li>
+         <li>RSPec</li>
+         <li>CapyBara</li>
+         <li>Selenium</li>
+        </ul>
+    </div>
+    <div class="s">
+     <ul>
+         <li><h4>Skills</h4></li>
+         <li>Codekit</li>
+         <li>Github</li>
+         <li>Codepen</li>
+         <li>Gitlab</li>
+         <li>Terminal</li>
+     </ul>
+    </div>
+
+</section>
+<div class="img">
+    <img src="images/Vector4.png" alt="line" class="img" id="img1">
+</div>
+<div class="contact">
+    <ul>
+        <li> <a href="#"><img src="image/Vector.png" alt="Github" class="media"></a></li>
+        <li><a href="#"><img src="image/Icon-Linkedin.png" alt="in" class="media"></a></li>
+        <li><a href="#"><img src="image/angelist-1.png" alt="angelist" class="media"></a></li>
+        <li><a href="#"><img src="image/Icon-Twitter.png" alt="Twitter" class="media"></a></li>
+        <li><a href="#"><img src="image/Icon-Medium.png" alt="M" class="media"></a></li>
+    </ul>
+</div>
+
+<div class="arrow">
+    <div class="bar">
+        <h5>Work</h5>
+        <img src="image/arrow.svg" alt="down-arrow"/> 
+    </div>
+    <div class="bar, bar1">
+        <img src="image/arrow.svg" alt="down-arrow"/>
+        <h5>Work</h5> 
+    </div>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="images/Vector5.png" alt="bar" class="img" id="line">
+</div>
+<section class="work-section">
+    <div class="placeholder">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+    
+    <div class="background-image1">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```

Comment:
- [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.